server/controllers/BookController.js

```diff
@@ -75,6 +75,65 @@ class BookController {
       return responseMessage(response, 500, { message: error.message });
     }
   }
+
+  /**
+* Edit Books
+* @name editBook
+* @param {Object} request
+* @param {Object} response
+* @returns {JSON} object
+* @memberof BookController
+*/
+  static async editBook(request, response) {
+    const { id } = request.params;
+    const {
+      title, description, isbn, price, yearPublished, authorId
+    } = request.body;
+    try {
+      const books = await Book.findAll();
```

Comment:
If the database has a large number of books, this query here will be very intensive for the server. Why not query the database using the 'findOne' method and a filter to get 'availableBook' and 'availableIsbn'? That would be better